tests/test_jwud_service.py

```
@@ -0,0 +1,464 @@
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    ("mode", "src", "out"),
+    [
+        ("jwud_compress", "game.wud", "game.wux"),
+        ("jwud_decompress", "game.wux", "game.wud"),
+    ],
+)
+async def test_convert_moves_the_produced_file_onto_the_output_path(
+    tmp_path, monkeypatch, fake_binary, mode, src, out,
```

Ruff (0.16.1)

[warning] Unused function argument: `fake_binary`

(ARG001)